task_4/Toma.Lobanova/src/main/java/ua/od/hillel/calculator/Calculator.java

```diff
@@ -0,0 +1,44 @@
+package ua.od.hillel.calculator;
+
+public class Calculator {
+
+    public static void calculate(int firstOperand, int secondOperand, String operation) {
+		double result;
+        switch (operation) {
+            case "addition":
+                result = firstOperand + secondOperand;
+                break;
+            case "subtraction":
+                result = firstOperand - secondOperand;
+                break;
+            case "multiplication":
+                result = firstOperand * secondOperand;
+                break;
+            case "division": {
+                if (secondOperand != 0) {
+                    result = (double) firstOperand / secondOperand;
+                } else {
+                    System.out.println("You can't divide by zero");
+                    return;
+                }
+                break;
+            }
+            case "absolute":
+                result = Math.abs(firstOperand);
+                break;
+            case "remainder":
+                result = (double) firstOperand % secondOperand;
+                break;
+            default: {
+                System.out.println("Calculator can't do this operation");
+                return;
+            }
+        }
+        System.out.print("The first operand = " + firstOperand);
+        if (operation != "absolute") {
```

Comment:
You can't compare String the same way as primitives.
Since it is an object which holds the char[] array in its internals you should compare it with the "equals" method